test/unit/Exception/ApiCallFailedTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Test\Unit\Exception;
+
+use OnMoon\OpenApiServerBundle\Exception\ApiCallFailed;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \OnMoon\OpenApiServerBundle\Exception\ApiCallFailed
+ */
+final class ApiCallFailedTest extends TestCase
```

Comment:
Not all methods are tested
https://coveralls.io/builds/32500345/source?filename=src/Exception/ApiCallFailed.php